src/__tests__/int/consumer.int.spec.ts

```diff
@@ -1,10 +1,20 @@
-import { Subscribe } from "../..";
+import { Publish, Subscribe } from "../..";
 jest.unmock("kafkajs");
 
 describe("Consumer Integration Tests", () => {
-  const propertiesResponse = ["key", "message", "offset", "partition", "topic"];
+  beforeAll(async () => {
+    const publish = Publish();
+    await publish.send("topic1", "hello world");
```

Comment:
Change the topic name to a differente name from the publisher one, like "test.consumer.mock.topic"